packages/rancher-monitoring/generated-changes/patch/charts/k3sServer/templates/pushprox-clients-rbac.yaml.patch

```diff
@@ -0,0 +1,16 @@
+--- charts-original/charts/k3sServer/templates/pushprox-clients-rbac.yaml
++++ charts/charts/k3sServer/templates/pushprox-clients-rbac.yaml
+@@ -11,7 +11,12 @@
+   resourceNames:
+   - {{ template "pushProxy.client.name" . }}
+ {{- if and .Values.clients.https.enabled .Values.clients.https.useServiceAccountCredentials }}
+-- nonResourceURLs: ["/metrics"]
++- nonResourceURLs:
++  - "/metrics"
++  - "/metrics/cadvisor"
++  verbs: ["get"]
++- apiGroups: [""]
++  resources: ["nodes/metrics"]
```

Comment:
same comments as above; let's avoid patching this subchart if it could be a commonly used feature anyways.